pulp_rpm/app/models/advisory.py

```diff
@@ -468,6 +468,7 @@ def to_createrepo_c(self):
             pkg.restart_suggested = self.restart_suggested
         if self.sum:
             pkg.sum = self.sum
+        if self.sum_type:
             pkg.sum_type = self.sum_type
```

Comment:
This assignment fails if `self.sum_type` is `None`, which I assume only happens when migrated from Pulp 2, as I cannot find any way to extract a `None` value from createrepo_c on sync.